website/docs/r/batch_pool.html.markdown

```diff
@@ -102,6 +102,35 @@ EOF
 }
 ```
 
+## Example Usage with a Custom VM Image
```

Comment:
Could we move this to the example folder: `examples/batch/pool/custom-image` or such?

Reply:
done.